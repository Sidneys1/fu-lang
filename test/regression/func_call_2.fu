test2: u8() => 22;

test: u8() => test2() + 20;

main: u8() => test();
