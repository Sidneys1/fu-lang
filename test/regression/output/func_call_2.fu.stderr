x x x x x x Failed to lex `Identity`: Expected Colon
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "S:\fugly-compiler\fu\compiler\__main__.py", line 147, in <module>
    sys.exit(main(*sys.argv[1:]))
             ^^^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\__main__.py", line 129, in main
    binary.encode(output)
  File "S:\fugly-compiler\fu\virtual_machine\bytecode\structures\__init__.py", line 19, in encode
    for x in _to_bytes(self._encode()):
  File "S:\fugly-compiler\fu\virtual_machine\bytecode\structures\__init__.py", line 47, in _to_bytes
    for x in in_:
  File "S:\fugly-compiler\fu\virtual_machine\bytecode\structures\binary.py", line 114, in _encode
    buffer.write(_encode_u32(v[1]))
                 ^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\virtual_machine\bytecode\__init__.py", line 19, in _encode_struct
    return struct.pack(pack, vals)
           ^^^^^^^^^^^^^^^^^^^^^^^
struct.error: 'I' format requires 0 <= number <= 4294967295
